Few improvements

sakrist opened the pull request:
- Added few optimisations;
- Compatible with Objective-C;
- MacOS framework.

AlexLittlejohn commented on DelaunayTriangulationSwift/Vertex.swift:
```diff
@@ -6,46 +6,45 @@
-        return Array(Set(self))
+extension Vertex: Hashable {
+    public var hashValue: Int {
+        return Int((x+y)*10000000)
```
This method of hashing is more likely to result in collisions vs the previous method. 1+1 = 2+0 and so on.

sakrist replied:
you right, I did not think about it in this way :) But I think converting to string and then back is inefficient. Will think about it more.

AlexLittlejohn commented on DelaunayTriangulationSwift/Vertex.swift:
```diff
@@ -6,46 +6,45 @@
-    }
+public func ==(lhs: Vertex, rhs: Vertex) -> Bool {
+    return lhs.x == rhs.x && lhs.y == rhs.y
 }
```
This method of implementing the equatable extension feels dirty to me. The implementation is separate from the conformance to the protocol that requires it whereas in the previous code it was together.

sakrist commented:
I've replaced hash function with another interesting implementation. 
From here
https://codereview.stackexchange.com/a/148777/28941

Also moved equatable back.
